src/OrbitGl/include/OrbitGl/CallTreeView.h

```diff
@@ -181,27 +181,60 @@ class CallTreeUnwindErrorType : public CallTreeNode {
       : CallTreeNode{parent}, error_type_{error_type} {
     ORBIT_CHECK(error_type != orbit_client_data::CallstackType::kComplete);
   }
+  ~CallTreeUnwindErrorType() override = default;
 
   [[nodiscard]] orbit_client_data::CallstackType error_type() const { return error_type_; }
 
  private:
   orbit_client_data::CallstackType error_type_;
 };
 
-class CallTreeView : public CallTreeNode {
+class CallTreeRoot : public CallTreeNode {
+ public:
+  CallTreeRoot() : CallTreeNode(nullptr) {}
+  ~CallTreeRoot() override = default;
+};
+
+class CallTreeView {
  public:
   [[nodiscard]] static std::unique_ptr<CallTreeView> CreateTopDownViewFromPostProcessedSamplingData(
       const orbit_client_data::PostProcessedSamplingData& post_processed_sampling_data,
-      const orbit_client_data::ModuleManager& module_manager,
-      const orbit_client_data::CaptureData& capture_data);
+      const orbit_client_data::ModuleManager* module_manager,
+      const orbit_client_data::CaptureData* capture_data);
 
   [[nodiscard]] static std::unique_ptr<CallTreeView>
   CreateBottomUpViewFromPostProcessedSamplingData(
       const orbit_client_data::PostProcessedSamplingData& post_processed_sampling_data,
-      const orbit_client_data::ModuleManager& module_manager,
-      const orbit_client_data::CaptureData& capture_data);
+      const orbit_client_data::ModuleManager* module_manager,
+      const orbit_client_data::CaptureData* capture_data);
+
+  CallTreeView(std::unique_ptr<CallTreeRoot> call_tree_root,
+               const orbit_client_data::ModuleManager* module_manager,
+               const orbit_client_data::CaptureData* capture_data)
+      : call_tree_root_{std::move(call_tree_root)},
+        module_manager_{module_manager},
+        capture_data_{capture_data} {
+    ORBIT_CHECK(call_tree_root_ != nullptr);
+  }
+
+  [[nodiscard]] const CallTreeRoot* GetRootCallTreeNode() const { return call_tree_root_.get(); }
```

Comment:
Now that the `CallTreeRoot` class has been added, I propose yet another name: `GetCallTreeRoot`.

Reply:
Done.